packages/math-input/src/components/key-translator.ts

```diff
@@ -0,0 +1,182 @@
+import Key from "../data/keys";
+import {MathQuillInterface} from "../types";
+
+enum ActionType {
+    WRITE = "write",
+    CMD = "cmd",
+    KEYSTROKE = "keystroke",
+    MQ_END = 0,
+}
+
+type MathQuillCallback = (mathQuill: MathQuillInterface) => void;
+
+function getGenericCallback(
```

Comment:
**nit**: I think of this as `buildLiteralCallback`.  Here's my possibly pedantic reasoning.

`build` -> We are constructing a callback, not getting or accessing it.
`Literal` -> It _is_ generic, but the literal value `str` that we're passing in is literally the action we're creating the action for. No translation. 

🤷‍♂️   Maybe the bike shed should be taupe.

Reply:
Unless you feel strongly about it, what if we met in the middle with `buildGenericCallback`? When I think "literal" I think "array literal", "object literal", and "string literal" so it's kind of confusing to me since this is returning a function.

Reply:
I'll leave it for you to decide on the final naming. My thinking on "literal" was that we pass the literal value and that literal value is sent to Mathquill directly (no translation). But no worries either way. :)